decisions/log/0031-mandatory-to-optional-attributes.md

```diff
@@ -0,0 +1,39 @@
+# Changing 5 attributes from Mandatory to Optional
+
+Date: 2024-03-13
+
+## Status
+
+Proposed
+
+## Context
+
+The PACT Technical Specifications v2.0 have 28 mandatory fields. Feedback was raised by the PACT Community that some fields should be reconsidered whether they should be mandatory. The five fields below were ultimately determined to be in-scope for this decision. Other mandatory attributes to be considered to be made optional can be considered in a separate ADR.
```

Comment:
I would propose omitting the minor version specific sentence, for two reasons: (i) it might raise questions as to why the minor version mentioned is not the latest; (ii) it would have to be updated after 2.2 release. 
```suggestion
Feedback was raised by the PACT Community that some mandatory fields should be reconsidered whether they should be mandatory. The five fields below were ultimately determined to be in-scope for this decision. Other mandatory attributes to be considered to be made optional can be considered in a separate ADR.
```